src/services/workspace-client/devWorkspaceClient.ts

```diff
@@ -31,6 +31,8 @@ export interface IStatusUpdate {
   workspaceId: string;
 }
 
+export const DEVWORKSPACE_NEXT_START_ANNOTATION = 'che.eclipse.org/dashboard-next-run';
```

Comment:
```suggestion
/**
 * DevWorkspace Configuration that is postponed till next start to avoid devworkspace restart.
 */
export const DEVWORKSPACE_NEXT_START_ANNOTATION = 'che.eclipse.org/next-start-cfg';
```
^ I assume it should be respected by any client, like chectl.
If we want to make it component specific, it would be `dashboard.che.eclipse.org/...` I think. Like openshift console has `console.openshift.io`

Reply:
I wonder if dashboard should flush such delayed configuration to already stopped workspaces? or it would just make stuff more complicated?

Reply:
Do you mean when the workspace is stopping then if there is something in the configuration apply it directly to the devworkspace then delete the annotation? It might make it a bit more complicated.

When you edit a stopped devworkspace in the dashboard and then click save it will automatically update the devworkspace directly instead of adding the annotation

Reply:
I meant flush annotation value to config as soon as possible, when Dashboard detect stopped workspace with the annotation.

It may help to avoid config lose when users edit config with kubectl, when it has dashboard annotation. Then Dashboard will override their changes with next run.
But it's minor, let's discuss and track it separately.